src/cmr/concepts/acl.js

```diff
@@ -0,0 +1,36 @@
+import AclConcept from './aclConcept'
+
+export default class Acl extends AclConcept {
+  /**
+   * Instantiates an Acl object
```

Comment:
"Instantiates an ACL object from the CMR API" or something a bit more descriptive.